shared/chat/conversation/list-area/normal/index.desktop.js

```diff
@@ -55,6 +55,20 @@ class Thread extends React.PureComponent<Props, State> {
     }
   }
 
+  _scrollDown = () => {
```

Comment:
This behavior is slightly different from a 'normal' scroll, e.g. it jumps instantly instead of smoothly scrolling. But I tried for a while to 'forward' the PageUp/Down keypress to the right control without success.

I took a look at what Slack did, and it's similar to this, anyway.